Unexpected schema domains in `lm8` snapshots

Latest `prod` build fails IT due to LungMap catalogs referencing invalid schema domains.
https://gitlab.azul.data.humancellatlas.org/ucsc/azul/-/jobs/73926
```
======================================================================
ERROR: test_indexing (integration_test.IndexingIntegrationTest.test_indexing)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builds/ucsc/azul/test/azul_test_case.py", line 268, in wrapped
    return method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/builds/ucsc/azul/test/integration_test.py", line 444, in test_indexing
    _wait_for_indexer()
  File "/builds/ucsc/azul/test/integration_test.py", line 408, in _wait_for_indexer
    self.azul_client.wait_for_indexer()
  File "/builds/ucsc/azul/src/azul/azulclient.py", line 501, in wait_for_indexer
    self.queues.wait_to_stabilize()
  File "/builds/ucsc/azul/src/azul/queues.py", line 276, in wait_to_stabilize
    raise Exception('The queues have stalled', final_length)
Exception: ('The queues have stalled', 2)
======================================================================
FAIL: test_can_bundle_configured_catalogs (integration_test.CanBundleScriptIntegrationTest.test_can_bundle_configured_catalogs) (catalog='lm7-it', repository=Config.Catalog.Plugin(name='tdr_hca'))
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builds/ucsc/azul/test/integration_test.py", line 1841, in test_can_bundle_configured_catalogs
    self._test_catalog(catalog)
  File "/builds/ucsc/azul/test/integration_test.py", line 1795, in _test_catalog
    self._can_bundle(fqid, output_dir=d)
  File "/builds/ucsc/azul/test/integration_test.py", line 1886, in _can_bundle
    return self._can_bundle_main(args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/builds/ucsc/azul/scripts/can_bundle.py", line 73, in main
    bundle = fetch_bundle(args.source, fqid_fields)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/builds/ucsc/azul/scripts/can_bundle.py", line 104, in fetch_bundle
    bundle = plugin.fetch_bundle(fqid)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/builds/ucsc/azul/src/azul/plugins/repository/tdr.py", line 202, in fetch_bundle
    bundle = self._emulate_bundle(bundle_fqid)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/builds/ucsc/azul/src/azul/plugins/repository/tdr_hca/__init__.py", line 347, in _emulate_bundle
    bundle.add_entity(entity=entity,
  File "/builds/ucsc/azul/src/azul/plugins/repository/tdr_hca/__init__.py", line 197, in add_entity
    Entity.validate_described_by(descriptor)
  File "/builds/ucsc/azul/src/humancellatlas/data/metadata/api.py", line 132, in validate_described_by
    assert described_by.netloc in valid_schema_domains, R(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: R('Unexpected schema domain', furl('https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/lungmap-phantom-files/json_schema/system/file_descriptor.json'), ['schema.humancellatlas.org'])
======================================================================
FAIL: test_can_bundle_configured_catalogs (integration_test.CanBundleScriptIntegrationTest.test_can_bundle_configured_catalogs) (catalog='lm8-it', repository=Config.Catalog.Plugin(name='tdr_hca'))
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builds/ucsc/azul/test/integration_test.py", line 1841, in test_can_bundle_configured_catalogs
    self._test_catalog(catalog)
  File "/builds/ucsc/azul/test/integration_test.py", line 1795, in _test_catalog
    self._can_bundle(fqid, output_dir=d)
  File "/builds/ucsc/azul/test/integration_test.py", line 1886, in _can_bundle
    return self._can_bundle_main(args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/builds/ucsc/azul/scripts/can_bundle.py", line 73, in main
    bundle = fetch_bundle(args.source, fqid_fields)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/builds/ucsc/azul/scripts/can_bundle.py", line 104, in fetch_bundle
    bundle = plugin.fetch_bundle(fqid)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/builds/ucsc/azul/src/azul/plugins/repository/tdr.py", line 202, in fetch_bundle
    bundle = self._emulate_bundle(bundle_fqid)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/builds/ucsc/azul/src/azul/plugins/repository/tdr_hca/__init__.py", line 347, in _emulate_bundle
    bundle.add_entity(entity=entity,
  File "/builds/ucsc/azul/src/azul/plugins/repository/tdr_hca/__init__.py", line 197, in add_entity
    Entity.validate_described_by(descriptor)
  File "/builds/ucsc/azul/src/humancellatlas/data/metadata/api.py", line 132, in validate_described_by
    assert described_by.netloc in valid_schema_domains, R(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: R('Unexpected schema domain', furl('https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/lungmap-phantom-files/json_schema/system/file_descriptor.json'), ['schema.humancellatlas.org'])
Slowest test durations
```

Furthermore, the `test_indexing` also relate to the same assertion:

```
[ERROR] AssertionError: R('Unexpected schema domain', furl('https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/lungmap-phantom-files/json_schema/system/file_descriptor.json'), ['schema.humancellatlas.org'])
Traceback (most recent call last):
  File "/var/task/azul/chalice.py", line 190, in patched_event_source_handler
    return old_handler(self_, event, context)
  File "/var/task/chalice/app.py", line 1761, in __call__
    return self.handler(event_obj)
  File "/var/task/app.py", line 193, in contribute
    app.index_controller.contribute(event)
  File "/var/task/azul/indexer/index_controller.py", line 175, in contribute
    contributions, replicas = self.transform(catalog, notification, delete)
  File "/var/task/azul/indexer/index_controller.py", line 225, in transform
    bundle = service.fetch_bundle(catalog, bundle_fqid)
  File "/var/task/azul/indexer/index_service.py", line 198, in fetch_bundle
    return plugin.fetch_bundle(bundle_fqid)
  File "/var/task/azul/plugins/repository/tdr.py", line 202, in fetch_bundle
    bundle = self._emulate_bundle(bundle_fqid)
  File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 347, in _emulate_bundle
    bundle.add_entity(entity=entity,
  File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 197, in add_entity
    Entity.validate_described_by(descriptor)
  File "/var/task/humancellatlas/data/metadata/api.py", line 132, in validate_described_by
    assert described_by.netloc in valid_schema_domains, R(
```


--- 

After a targeted reindex, the following snapshots failed to index in the following catalogs,

`lm7`:
`lungmap_prod_00f056f273ff43ac97ff69ca10e38c89__20220308_20231207_lm4`
`lungmap_prod_6511b041b11e4ccf85932b40148c437e__20240326_20240326_lm6`
`lungmap_prod_20037472ea1d4ddb9cd356a11a6f0f76__20220307_20231207_lm4`
`lungmap_prod_2620497955a349b28d2b53e0bdfcb176__20220308_20231207_lm4`
`lungmap_prod_1977dc4784144263a8706b0f207d8ab3__20240206_20240626_lm7`
`lungmap_prod_f899709cae2c4bb988f0131142e6c7ec__20220310_20231207_lm4`
`lungmap_prod_fdadee7e209745d5bf81cc280bd8348e__20240206_20240626_lm7`
`lungmap_prod_4ae8c5c91520437198276935661f6c84__20231004_20231207_lm4`


`lm8`:
`lungmap_prod_6511b041b11e4ccf85932b40148c437e__20240326_20240326_lm6`
`lungmap_prod_2620497955a349b28d2b53e0bdfcb176__20220308_20231207_lm4`
`lungmap_prod_00f056f273ff43ac97ff69ca10e38c89__20220308_20231207_lm4`

`lm7` failures:

**CloudWatch Logs Insights**    
region: us-east-1    
log-group-names: /aws/lambda/azul-indexer-prod-contribute    
start-time: -3600s    
end-time: 0s    
query-string:
  ```
  fields @timestamp, @message
| filter @message like /Worker failed to handle message/
# | filter @requestId like /e249333f|f737270b/
| sort @timestamp asc
| limit 10000
  ```
---
| @timestamp | @message |
| --- | --- |
| 2025-03-08 07:09:18.148 | [WARNING] 2025-03-08T07:09:18.147Z 28c88fba-8853-556e-be29-efff670396f4 azul.indexer.index_controller Worker failed to handle message {'action': 'add', 'notification': {'transaction_id': '2041e1ad-68c8-4c25-8960-afbaec068957', 'bundle_fqid': {'uuid': 'a078a6cb-a72a-305c-80df-cf35aedd01ff', 'version': '2023-10-09T12:05:57.028794Z', 'source': {'id': 'ccd90933-7889-458b-9cef-a0fe5451867e', 'spec': 'tdr:bigquery:gcp:datarepo-3d684ccd:lungmap_prod_00f056f273ff43ac97ff69ca10e38c89__20220308_20231207_lm4:/0'}}}, 'catalog': 'lm7'}. Traceback (most recent call last):   File "/var/task/azul/indexer/index_controller.py", line 175, in contribute     contributions, replicas = self.transform(catalog, notification, delete)                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_controller.py", line 225, in transform     bundle = service.fetch_bundle(catalog, bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_service.py", line 198, in fetch_bundle     return plugin.fetch_bundle(bundle_fqid)            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr.py", line 202, in fetch_bundle     bundle = self._emulate_bundle(bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 347, in _emulate_bundle     bundle.add_entity(entity=entity,   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 197, in add_entity     Entity.validate_described_by(descriptor)   File "/var/task/humancellatlas/data/metadata/api.py", line 132, in validate_described_by     assert described_by.netloc in valid_schema_domains, R(            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ AssertionError: R('Unexpected schema domain', furl('https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/lungmap-phantom-files/json_schema/system/file_descriptor.json'), ['schema.humancellatlas.org']) |
| 2025-03-08 07:09:20.799 | [WARNING] 2025-03-08T07:09:20.797Z 6f3e90ae-705a-5a75-8205-f0eb9f0909ef azul.indexer.index_controller Worker failed to handle message {'action': 'add', 'notification': {'transaction_id': 'a678dfbc-7c93-48ba-97e0-4d3e7b57ab6f', 'bundle_fqid': {'uuid': '5a6c7366-db03-3d63-b995-fa11fd68846d', 'version': '2023-12-18T21:37:54.723662Z', 'source': {'id': '996b613b-33c3-4484-afdb-a53ddc5cd20b', 'spec': 'tdr:bigquery:gcp:datarepo-c3ad47d2:lungmap_prod_6511b041b11e4ccf85932b40148c437e__20240326_20240326_lm6:/0'}}}, 'catalog': 'lm7'}. Traceback (most recent call last):   File "/var/task/azul/indexer/index_controller.py", line 175, in contribute     contributions, replicas = self.transform(catalog, notification, delete)                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_controller.py", line 225, in transform     bundle = service.fetch_bundle(catalog, bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_service.py", line 198, in fetch_bundle     return plugin.fetch_bundle(bundle_fqid)            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr.py", line 202, in fetch_bundle     bundle = self._emulate_bundle(bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 347, in _emulate_bundle     bundle.add_entity(entity=entity,   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 197, in add_entity     Entity.validate_described_by(descriptor)   File "/var/task/humancellatlas/data/metadata/api.py", line 132, in validate_described_by     assert described_by.netloc in valid_schema_domains, R(            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ AssertionError: R('Unexpected schema domain', furl('https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/lungmap-phantom-files/json_schema/system/file_descriptor.json'), ['schema.humancellatlas.org']) |
| 2025-03-08 07:09:24.135 | [WARNING] 2025-03-08T07:09:24.133Z 11bbaaeb-5844-5ac9-a95c-caaeb7178953 azul.indexer.index_controller Worker failed to handle message {'action': 'add', 'notification': {'transaction_id': '1347ae26-e0c7-4930-af04-cff3070fe756', 'bundle_fqid': {'uuid': 'c97b7890-07fa-345f-a47e-4134536f9367', 'version': '2022-02-22T18:38:07.919092Z', 'source': {'id': 'a35e4770-1131-4dc3-955d-f0e7f526dc21', 'spec': 'tdr:bigquery:gcp:datarepo-43d1f2cb:lungmap_prod_20037472ea1d4ddb9cd356a11a6f0f76__20220307_20231207_lm4:/0'}}}, 'catalog': 'lm7'}. Traceback (most recent call last):   File "/var/task/azul/indexer/index_controller.py", line 175, in contribute     contributions, replicas = self.transform(catalog, notification, delete)                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_controller.py", line 225, in transform     bundle = service.fetch_bundle(catalog, bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_service.py", line 198, in fetch_bundle     return plugin.fetch_bundle(bundle_fqid)            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr.py", line 202, in fetch_bundle     bundle = self._emulate_bundle(bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 347, in _emulate_bundle     bundle.add_entity(entity=entity,   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 197, in add_entity     Entity.validate_described_by(descriptor)   File "/var/task/humancellatlas/data/metadata/api.py", line 132, in validate_described_by     assert described_by.netloc in valid_schema_domains, R(            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ AssertionError: R('Unexpected schema domain', furl('https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/lungmap-phantom-files/json_schema/system/file_descriptor.json'), ['schema.humancellatlas.org']) |
| 2025-03-08 07:09:25.025 | [WARNING] 2025-03-08T07:09:25.023Z 4881a136-c111-5ace-9302-c1732e421954 azul.indexer.index_controller Worker failed to handle message {'action': 'add', 'notification': {'transaction_id': '755c31ec-80f3-485c-8ebf-5c1a33ded390', 'bundle_fqid': {'uuid': '153f5c7a-593e-37fc-aa56-4870057f27c7', 'version': '2021-10-19T17:48:16.442739Z', 'source': {'id': 'ad716812-8644-4c69-a630-7929cef5040e', 'spec': 'tdr:bigquery:gcp:datarepo-91587240:lungmap_prod_2620497955a349b28d2b53e0bdfcb176__20220308_20231207_lm4:/0'}}}, 'catalog': 'lm7'}. Traceback (most recent call last):   File "/var/task/azul/indexer/index_controller.py", line 175, in contribute     contributions, replicas = self.transform(catalog, notification, delete)                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_controller.py", line 225, in transform     bundle = service.fetch_bundle(catalog, bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_service.py", line 198, in fetch_bundle     return plugin.fetch_bundle(bundle_fqid)            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr.py", line 202, in fetch_bundle     bundle = self._emulate_bundle(bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 347, in _emulate_bundle     bundle.add_entity(entity=entity,   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 197, in add_entity     Entity.validate_described_by(descriptor)   File "/var/task/humancellatlas/data/metadata/api.py", line 132, in validate_described_by     assert described_by.netloc in valid_schema_domains, R(            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ AssertionError: R('Unexpected schema domain', furl('https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/lungmap-phantom-files/json_schema/system/file_descriptor.json'), ['schema.humancellatlas.org']) |
| 2025-03-08 07:09:28.685 | [WARNING] 2025-03-08T07:09:28.683Z 0127ba04-576f-5ad7-adec-da41b836494d azul.indexer.index_controller Worker failed to handle message {'action': 'add', 'notification': {'transaction_id': '460e7e36-6c40-4ddc-ad45-ddb85a6ad134', 'bundle_fqid': {'uuid': '448cbba2-81d8-390e-b5b0-3547a4722b85', 'version': '2024-06-18T17:13:23.159667Z', 'source': {'id': '6df22bd3-1fc5-4689-8f4d-31fd6a5f687f', 'spec': 'tdr:bigquery:gcp:datarepo-2555a3ee:lungmap_prod_1977dc4784144263a8706b0f207d8ab3__20240206_20240626_lm7:/0'}}}, 'catalog': 'lm7'}. Traceback (most recent call last):   File "/var/task/azul/indexer/index_controller.py", line 175, in contribute     contributions, replicas = self.transform(catalog, notification, delete)                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_controller.py", line 225, in transform     bundle = service.fetch_bundle(catalog, bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_service.py", line 198, in fetch_bundle     return plugin.fetch_bundle(bundle_fqid)            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr.py", line 202, in fetch_bundle     bundle = self._emulate_bundle(bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 347, in _emulate_bundle     bundle.add_entity(entity=entity,   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 197, in add_entity     Entity.validate_described_by(descriptor)   File "/var/task/humancellatlas/data/metadata/api.py", line 132, in validate_described_by     assert described_by.netloc in valid_schema_domains, R(            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ AssertionError: R('Unexpected schema domain', furl('https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/lungmap-phantom-files/json_schema/system/file_descriptor.json'), ['schema.humancellatlas.org']) |
| 2025-03-08 07:09:29.459 | [WARNING] 2025-03-08T07:09:29.458Z 14b96183-5d60-5c65-a5c4-01c6b2396e87 azul.indexer.index_controller Worker failed to handle message {'action': 'add', 'notification': {'transaction_id': '839b8885-4b8e-4049-a80b-7902201aa94d', 'bundle_fqid': {'uuid': '31244706-60f8-3043-8db7-a39fd7081139', 'version': '2022-03-11T17:13:52.129998Z', 'source': {'id': '15324e53-b8d3-40b7-92fb-abfeb35e6515', 'spec': 'tdr:bigquery:gcp:datarepo-3f332829:lungmap_prod_f899709cae2c4bb988f0131142e6c7ec__20220310_20231207_lm4:/0'}}}, 'catalog': 'lm7'}. Traceback (most recent call last):   File "/var/task/azul/indexer/index_controller.py", line 175, in contribute     contributions, replicas = self.transform(catalog, notification, delete)                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_controller.py", line 225, in transform     bundle = service.fetch_bundle(catalog, bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_service.py", line 198, in fetch_bundle     return plugin.fetch_bundle(bundle_fqid)            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr.py", line 202, in fetch_bundle     bundle = self._emulate_bundle(bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 347, in _emulate_bundle     bundle.add_entity(entity=entity,   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 197, in add_entity     Entity.validate_described_by(descriptor)   File "/var/task/humancellatlas/data/metadata/api.py", line 132, in validate_described_by     assert described_by.netloc in valid_schema_domains, R(            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ AssertionError: R('Unexpected schema domain', furl('https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/lungmap-phantom-files/json_schema/system/file_descriptor.json'), ['schema.humancellatlas.org']) |
| 2025-03-08 07:09:30.039 | [WARNING] 2025-03-08T07:09:30.037Z 59872e7d-dd04-5ca8-af00-4e38325d6f32 azul.indexer.index_controller Worker failed to handle message {'action': 'add', 'notification': {'transaction_id': '7b3a5452-e5a4-4d62-b2a2-8ff52a29568b', 'bundle_fqid': {'uuid': '1928ae54-dbba-33e6-9e40-6a9b3fe8585f', 'version': '2024-06-25T15:12:06.304736Z', 'source': {'id': 'd6396258-03bd-42b6-8720-e2e47d8672b7', 'spec': 'tdr:bigquery:gcp:datarepo-43814140:lungmap_prod_fdadee7e209745d5bf81cc280bd8348e__20240206_20240626_lm7:/0'}}}, 'catalog': 'lm7'}. Traceback (most recent call last):   File "/var/task/azul/indexer/index_controller.py", line 175, in contribute     contributions, replicas = self.transform(catalog, notification, delete)                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_controller.py", line 225, in transform     bundle = service.fetch_bundle(catalog, bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_service.py", line 198, in fetch_bundle     return plugin.fetch_bundle(bundle_fqid)            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr.py", line 202, in fetch_bundle     bundle = self._emulate_bundle(bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 347, in _emulate_bundle     bundle.add_entity(entity=entity,   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 197, in add_entity     Entity.validate_described_by(descriptor)   File "/var/task/humancellatlas/data/metadata/api.py", line 132, in validate_described_by     assert described_by.netloc in valid_schema_domains, R(            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ AssertionError: R('Unexpected schema domain', furl('https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/lungmap-phantom-files/json_schema/system/file_descriptor.json'), ['schema.humancellatlas.org']) |
| 2025-03-08 07:09:30.249 | [WARNING] 2025-03-08T07:09:30.247Z a94e8c6e-64e4-5236-bcb7-ec2b8bbc81ac azul.indexer.index_controller Worker failed to handle message {'action': 'add', 'notification': {'transaction_id': '77b3a671-035e-46be-9299-4bb0e72a8149', 'bundle_fqid': {'uuid': '61e6eb8d-8914-3acc-9669-3c979557a55e', 'version': '2023-10-27T15:17:06.396825Z', 'source': {'id': '5f5f73b0-e6c9-444e-9ed2-fcc454485944', 'spec': 'tdr:bigquery:gcp:datarepo-252f2a7d:lungmap_prod_4ae8c5c91520437198276935661f6c84__20231004_20231207_lm4:/0'}}}, 'catalog': 'lm7'}. Traceback (most recent call last):   File "/var/task/azul/indexer/index_controller.py", line 175, in contribute     contributions, replicas = self.transform(catalog, notification, delete)                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_controller.py", line 225, in transform     bundle = service.fetch_bundle(catalog, bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_service.py", line 198, in fetch_bundle     return plugin.fetch_bundle(bundle_fqid)            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr.py", line 202, in fetch_bundle     bundle = self._emulate_bundle(bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 347, in _emulate_bundle     bundle.add_entity(entity=entity,   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 197, in add_entity     Entity.validate_described_by(descriptor)   File "/var/task/humancellatlas/data/metadata/api.py", line 132, in validate_described_by     assert described_by.netloc in valid_schema_domains, R(            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ AssertionError: R('Unexpected schema domain', furl('https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/lungmap-phantom-files/json_schema/system/file_descriptor.json'), ['schema.humancellatlas.org']) |
---


`lm8` failures:

**CloudWatch Logs Insights**    
region: us-east-1    
log-group-names: /aws/lambda/azul-indexer-prod-contribute    
start-time: -10800s    
end-time: 0s    
query-string:
  ```
  fields @timestamp, @message
| filter @message like /Worker failed to handle message/
| filter @message like /lm8/
# | filter @requestId like /e249333f|f737270b/
| sort @timestamp asc
| limit 10000
  ```
---
| @timestamp | @message |
| --- | --- |
| 2025-03-08 05:44:15.194 | [WARNING] 2025-03-08T05:44:15.193Z 24653c4d-9d4e-5de3-9afa-5c08462e1325 azul.indexer.index_controller Worker failed to handle message {'action': 'add', 'notification': {'transaction_id': 'ea190f94-4b08-4de1-bcdf-d1e076fbea5a', 'bundle_fqid': {'uuid': '5a6c7366-db03-3d63-b995-fa11fd68846d', 'version': '2023-12-18T21:37:54.723662Z', 'source': {'id': '996b613b-33c3-4484-afdb-a53ddc5cd20b', 'spec': 'tdr:bigquery:gcp:datarepo-c3ad47d2:lungmap_prod_6511b041b11e4ccf85932b40148c437e__20240326_20240326_lm6:/0'}}}, 'catalog': 'lm8'}. Traceback (most recent call last):   File "/var/task/azul/indexer/index_controller.py", line 175, in contribute     contributions, replicas = self.transform(catalog, notification, delete)                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_controller.py", line 225, in transform     bundle = service.fetch_bundle(catalog, bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_service.py", line 198, in fetch_bundle     return plugin.fetch_bundle(bundle_fqid)            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr.py", line 202, in fetch_bundle     bundle = self._emulate_bundle(bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 347, in _emulate_bundle     bundle.add_entity(entity=entity,   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 197, in add_entity     Entity.validate_described_by(descriptor)   File "/var/task/humancellatlas/data/metadata/api.py", line 132, in validate_described_by     assert described_by.netloc in valid_schema_domains, R(            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ AssertionError: R('Unexpected schema domain', furl('https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/lungmap-phantom-files/json_schema/system/file_descriptor.json'), ['schema.humancellatlas.org']) |
| 2025-03-08 05:44:19.506 | [WARNING] 2025-03-08T05:44:19.504Z 73efdcbf-f396-5c22-a352-bcfa28f792e4 azul.indexer.index_controller Worker failed to handle message {'action': 'add', 'notification': {'transaction_id': 'e5936602-b53a-4548-a489-7fba8fe69c41', 'bundle_fqid': {'uuid': '153f5c7a-593e-37fc-aa56-4870057f27c7', 'version': '2021-10-19T17:48:16.442739Z', 'source': {'id': 'ad716812-8644-4c69-a630-7929cef5040e', 'spec': 'tdr:bigquery:gcp:datarepo-91587240:lungmap_prod_2620497955a349b28d2b53e0bdfcb176__20220308_20231207_lm4:/0'}}}, 'catalog': 'lm8'}. Traceback (most recent call last):   File "/var/task/azul/indexer/index_controller.py", line 175, in contribute     contributions, replicas = self.transform(catalog, notification, delete)                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_controller.py", line 225, in transform     bundle = service.fetch_bundle(catalog, bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_service.py", line 198, in fetch_bundle     return plugin.fetch_bundle(bundle_fqid)            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr.py", line 202, in fetch_bundle     bundle = self._emulate_bundle(bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 347, in _emulate_bundle     bundle.add_entity(entity=entity,   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 197, in add_entity     Entity.validate_described_by(descriptor)   File "/var/task/humancellatlas/data/metadata/api.py", line 132, in validate_described_by     assert described_by.netloc in valid_schema_domains, R(            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ AssertionError: R('Unexpected schema domain', furl('https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/lungmap-phantom-files/json_schema/system/file_descriptor.json'), ['schema.humancellatlas.org']) |
| 2025-03-08 05:44:25.910 | [WARNING] 2025-03-08T05:44:25.909Z 71c528d2-15f8-5c06-8a03-4897ce9292c0 azul.indexer.index_controller Worker failed to handle message {'action': 'add', 'notification': {'transaction_id': '76625524-0dd1-4c16-a1dc-2aabb52a2713', 'bundle_fqid': {'uuid': 'a078a6cb-a72a-305c-80df-cf35aedd01ff', 'version': '2023-10-09T12:05:57.028794Z', 'source': {'id': 'ccd90933-7889-458b-9cef-a0fe5451867e', 'spec': 'tdr:bigquery:gcp:datarepo-3d684ccd:lungmap_prod_00f056f273ff43ac97ff69ca10e38c89__20220308_20231207_lm4:/0'}}}, 'catalog': 'lm8'}. Traceback (most recent call last):   File "/var/task/azul/indexer/index_controller.py", line 175, in contribute     contributions, replicas = self.transform(catalog, notification, delete)                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_controller.py", line 225, in transform     bundle = service.fetch_bundle(catalog, bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/indexer/index_service.py", line 198, in fetch_bundle     return plugin.fetch_bundle(bundle_fqid)            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr.py", line 202, in fetch_bundle     bundle = self._emulate_bundle(bundle_fqid)              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 347, in _emulate_bundle     bundle.add_entity(entity=entity,   File "/var/task/azul/plugins/repository/tdr_hca/__init__.py", line 197, in add_entity     Entity.validate_described_by(descriptor)   File "/var/task/humancellatlas/data/metadata/api.py", line 132, in validate_described_by     assert described_by.netloc in valid_schema_domains, R(            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ AssertionError: R('Unexpected schema domain', furl('https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/lungmap-phantom-files/json_schema/system/file_descriptor.json'), ['schema.humancellatlas.org']) |
---

--- a/deployments/prod/environment.py
+++ b/deployments/prod/environment.py
@@ -1303,10 +1303,13 @@ dcp45_sources = mkdict(dcp44_sources, 484, mkdelta([
     # @formatter:on
 ]))
 
-dcp46_sources = mkdict(dcp45_sources, 492, mkdelta([
+dcp46_sources = mkdict(dcp45_sources, 489, mkdelta([
+    # @formatter:off
     mksrc('bigquery', 'datarepo-6ecaa13f', 'hca_prod_031980e69f2b433a8f6e081bd9aad0a3__20250203_dcp2_20250211_dcp46'),
+    mksrc('bigquery', 'datarepo-2be1e691', 'hca_prod_0777b9ef91f3468b9deadb477437aa1a__20220330_dcp2_20230314_dcp25', pop),  # noqa E501
     mksrc('bigquery', 'datarepo-e8dc17ba', 'hca_prod_0cc58d0b17344e1d9113b32e52f75e36__20240531_dcp2_20250211_dcp46'),
     mksrc('bigquery', 'datarepo-88134e98', 'hca_prod_102018327c7340339b653ef13d81656a__20220213_dcp2_20250211_dcp46'),
+    mksrc('bigquery', 'datarepo-4e5e9f9b', 'hca_prod_2043c65a1cf84828a6569e247d4e64f1__20220111_dcp2_20220120_dcp12', pop),  # noqa E501
     mksrc('bigquery', 'datarepo-c3a6307c', 'hca_prod_2433c1e45a1246a5b2d5d3554a7694f2__20241205_dcp2_20250211_dcp46'),
     mksrc('bigquery', 'datarepo-3928cdc0', 'hca_prod_24d0dbbc54eb49048141934d26f1c936__20220303_dcp2_20250211_dcp46'),
     mksrc('bigquery', 'datarepo-996ef22b', 'hca_prod_3089d311f9ed44ddbb10397059bad4dc__20220111_dcp2_20250211_dcp46'),
@@ -1324,6 +1327,7 @@ dcp46_sources = mkdict(dcp45_sources, 492, mkdelta([
     mksrc('bigquery', 'datarepo-bab0d9fc', 'hca_prod_6f89a7f38d4a4344aa4feccfe7e91076__20220213_dcp2_20250211_dcp46'),
     mksrc('bigquery', 'datarepo-b2c70f75', 'hca_prod_750b455ae3cf472195818609a6c9d561__20240327_dcp2_20250211_dcp46'),
     mksrc('bigquery', 'datarepo-ec282ab2', 'hca_prod_783c9952a4ae4106a6ce56f20ce27f88__20220117_dcp2_20250211_dcp46'),
+    mksrc('bigquery', 'datarepo-ff62a88c', 'hca_prod_7b393e4d65bc4c03b402aae769299329__20220519_dcp2_20230314_dcp25', pop),  # noqa E501
     mksrc('bigquery', 'datarepo-a2ed4715', 'hca_prod_7f980afc4e014b9ca9a1f50e17fce8c2__20250203_dcp2_20250211_dcp46'),
     mksrc('bigquery', 'datarepo-644d80c0', 'hca_prod_84d1697fd4af42c29a5037fb5842c586__20250203_dcp2_20250211_dcp46'),
     mksrc('bigquery', 'datarepo-226acb8e', 'hca_prod_9dd91b6e7c6249d3a3d474f603deffdb__20240903_dcp2_20250211_dcp46'),
@@ -1335,6 +1339,7 @@ dcp46_sources = mkdict(dcp45_sources, 492, mkdelta([
     mksrc('bigquery', 'datarepo-b72037cb', 'hca_prod_e255b1c611434fa683a8528f15b41038__20220330_dcp2_20250211_dcp46'),
     mksrc('bigquery', 'datarepo-1514f0eb', 'hca_prod_e579d0f418ef4d4290fa4d2afb60a862__20250203_dcp2_20250211_dcp46'),
     mksrc('bigquery', 'datarepo-753fc76a', 'hca_prod_f77290ae0d7b4239b0fe3cf2c9e8858d__20250203_dcp2_20250211_dcp46'),
+    # @formatter:on
 ]))
 
 lungmap_sources = mkdict({}, 3, mkdelta([

--- a/deployments/prod/environment.py
+++ b/deployments/prod/environment.py
@@ -1434,7 +1434,6 @@ def env() -> Mapping[str, Optional[str]]:
                                                     repository=dict(name='tdr_hca')),
                                        sources=mklist(sources))
             for atlas, catalog, sources in [
-                ('hca', 'dcp45', dcp45_sources),
                 ('hca', 'dcp46', dcp46_sources),
                 ('lungmap', 'lm8', lm8_sources)
             ] for suffix, internal in [

--- a/deployments/prod/environment.py
+++ b/deployments/prod/environment.py
@@ -1431,7 +1431,6 @@ def env() -> Mapping[str, Optional[str]]:
             for atlas, catalog, sources in [
                 ('hca', 'dcp45', dcp45_sources),
                 ('hca', 'dcp46', dcp46_sources),
-                ('lungmap', 'lm7', lm7_sources),
                 ('lungmap', 'lm8', lm8_sources)
             ] for suffix, internal in [
                 ('', False),

--- a/deployments/prod/environment.py
+++ b/deployments/prod/environment.py
@@ -1368,12 +1368,18 @@ lm6_sources = mkdict(lm4_sources, 8, mkdelta([
     mksrc('bigquery', 'datarepo-c3ad47d2', 'lungmap_prod_6511b041b11e4ccf85932b40148c437e__20240326_20240326_lm6'),
 ]))
 
-lm7_sources = mkdict(lm6_sources, 10, mkdelta([
-    mksrc('bigquery', 'datarepo-2555a3ee', 'lungmap_prod_1977dc4784144263a8706b0f207d8ab3__20240206_20240626_lm7'),
-    mksrc('bigquery', 'datarepo-43814140', 'lungmap_prod_fdadee7e209745d5bf81cc280bd8348e__20240206_20240626_lm7')
+lm7_sources = mkdict(lm6_sources, 2, mkdelta([
+    # @formatter:off
+    mksrc('bigquery', 'datarepo-3d684ccd', 'lungmap_prod_00f056f273ff43ac97ff69ca10e38c89__20220308_20231207_lm4', pop),  # noqa E501
+    mksrc('bigquery', 'datarepo-43d1f2cb', 'lungmap_prod_20037472ea1d4ddb9cd356a11a6f0f76__20220307_20231207_lm4', pop),  # noqa E501
+    mksrc('bigquery', 'datarepo-91587240', 'lungmap_prod_2620497955a349b28d2b53e0bdfcb176__20220308_20231207_lm4', pop),  # noqa E501
+    mksrc('bigquery', 'datarepo-252f2a7d', 'lungmap_prod_4ae8c5c91520437198276935661f6c84__20231004_20231207_lm4', pop),  # noqa E501
+    mksrc('bigquery', 'datarepo-c3ad47d2', 'lungmap_prod_6511b041b11e4ccf85932b40148c437e__20240326_20240326_lm6', pop),  # noqa E501
+    mksrc('bigquery', 'datarepo-3f332829', 'lungmap_prod_f899709cae2c4bb988f0131142e6c7ec__20220310_20231207_lm4', pop),  # noqa E501
+    # @formatter:on
 ]))
 
-lm8_sources = mkdict(lm7_sources, 12, mkdelta([
+lm8_sources = mkdict(lm7_sources, 9, mkdelta([
     mksrc('bigquery', 'datarepo-2b15227b', 'lungmap_prod_1977dc4784144263a8706b0f207d8ab3__20240206_20241002_lm8'),
     mksrc('bigquery', 'datarepo-c9158593', 'lungmap_prod_20037472ea1d4ddb9cd356a11a6f0f76__20220307_20241002_lm8'),
     mksrc('bigquery', 'datarepo-35a6d7ca', 'lungmap_prod_3a02d15f9c6a4ef7852b4ddec733b70b__20241001_20241002_lm8'),

